config/kubernetes/org.yaml

```diff
@@ -191,6 +191,7 @@ members:
 - carlory
 - carlosedp
 - carolynvs
+- carolinar7
```

Comment:
The org membership list is alpha sorted^^;;; Could you move this up?

Reply:
Updated!